installer/resources/pacbot_app/files/DB.sql

```diff
@@ -1460,6 +1460,7 @@ INSERT IGNORE INTO `cf_Policy` (`policyId`,`policyName`,`policyDesc`,`resolution
 INSERT IGNORE INTO `cf_Policy` (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`) VALUES ('Aws_athena_query_results_should_be_encrypted_version-1','AwsAthenQueryResultsEncryption','Ensure that AWS Athena query results stored in Amazon S3 are encrypted at rest.','To enable encryption, enable the encrypt query results in settings and choose the Customer Master Key created.',NULL,'version-1',NULL,NULL,'2022-06-13','2022-06-13');
 INSERT IGNORE INTO `cf_Policy` (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`) VALUES ('AWS_Support_Access_Role_version-1','IAM role for AWS support access','Create a dedicated support role to manage incidents with AWS Support','If there is no role created, then create a separate support role and attach the policy to the role. Users or group should use this role for support access.',NULL,'version-1',NULL,NULL,'2022-08-31','2022-08-31');
 INSERT IGNORE INTO `cf_Policy` (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('RDSDBEnableAutoMinorVersionUpgrade_version-1','RDS Auto Minor Version Upgrade should be enabled','RDS database instances should have the Auto Minor Version Upgrade flag enabled to receive minor engine upgrades automatically.','If the DB instances have not enabled the flag for auto minor version upgrade.',NULL,'version-1',NULL,NULL,'2022-09-23','2022-09-23','https://github.com/PaladinCloud/CE/wiki/AWS-Policy#enable-rds-auto-minor-version-upgrade');
+INSERT IGNORE INTO `cf_Policy` (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('RemoveRootUserAccessKey_version-1','Root Account access keys should be removed','Checking access keys associated with the 'root' user account are removed','Delete Account access keys for Root user',NULL,'version-1',NULL,NULL,'2022-05-12','2022-05-12','https://github.com/PaladinCloud/CE/wiki/AWS-Policy#remove-root-user-account-access-key');
```

Comment:
The query is not valid, please escape the quotations in policyDesc

Reply:
Issue resolved